models/tenant.rb

```diff
@@ -0,0 +1,34 @@
+# require_relative "data"
+# require_relative "models/apartment"
+# require_relative "models/tenant"
+# apartments = data[:apartments]
+# tenants = data[:tenants]
+# ruby_apartments = []
+# ruby_tenants = []
+
+class Tenant
+  attr_accessor :id, :name, :age, :apartment_id
+
+  def initialize(id, name, age, apartment_id)
+    @id = id
+    @name = name
+    @age = age
+    @apartment_id = apartment_id
+  end
+
+  def id
```

Comment:
Since you have `attr_accessor` above, these getter methods are necessary any more!